src/client/Microsoft.Identity.Client/Platforms/iOS/SystemWebview/SystemWebUI.cs

```diff
@@ -65,6 +65,12 @@ public void Authenticate(Uri authorizationUri, Uri redirectUri, RequestContext r
                             }
                         });
 
+                    // iOS 13 requires a PresentationContextProvider
+                    if (UIDevice.CurrentDevice.CheckSystemVersion(13, 0))
```

Comment:
>13 [](start = 66, length = 2)

What does CheckSystemVersion actually do? Is there a way this can be v13 and above?

Reply:
@henrik-me that's what it does. here is [documentation](https://docs.microsoft.com/en-us/dotnet/api/UIKit.UIDevice.CheckSystemVersion?view=xamarin-ios-sdk-12#UIKit_UIDevice_CheckSystemVersion_System_Int32_System_Int32_)

Reply:
lol

---
In reply to: [331427726](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/1414#discussion_r331427726) [](ancestors = 331427726)